Bonsai.Harp.CF/SyringePumpEvent.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Bonsai.Expressions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reactive.Linq;
+using TResult = System.String;
+using System.ComponentModel;
+
+namespace Bonsai.Harp.CF
+{
+    public enum SyringePumpEventType: byte
+    {
+        /* Event: STEP_STATE */
+        Step = 0,
+
+        /* Event: DIR_STATE */
+        Direction,
+
+        /* Event: SW_FORWARD_STATE */
+        SwitchForward,
+
+        /* Event: SW_REVERSE_STATE */
+        SwitchReverse,
+
+        /* Event: INPUT_STATE */
+        Input
+    }
+
+    [TypeDescriptionProvider(typeof(DeviceTypeDescriptionProvider<WearEvent>))]
```

Comment:
The type of `DeviceTypeDescriptionProvider` should be `SyringePumpEvent` instead of `WearEvent` to ensure the node descriptions are mapped correctly in the editor. The impact of this might not be obvious until the next release of the Bonsai editor, where the operator description will update in real-time with changes to the value of the `Type` property.